nixos/modules/security/rtkit.nix

```
@@ -35,9 +35,12 @@ with lib;
     services.dbus.packages = [ pkgs.rtkit ];
 
     users.users.rtkit =
-      { uid = config.ids.uids.rtkit;
+      {
+        uid = config.ids.uids.rtkit;
+        group = "rtkit";
         description = "RealtimeKit daemon";
       };
+    users.groups.rtkit.gid = config.ids.gids.rtkit;
```

Re https://github.com/NixOS/nixpkgs/pull/133166#discussion_r689806816, you can change this to
```suggestion
    users.groups.rtkit = {};
```

Which will work just as well as the current version of this PR, but without requiring a static gid (so the `rtkit` entry in `ids.nix` can be deleted). This is exactly what NixOS/rfcs#52 is about, to not use static uid/gid's, which this PR goes against.

But also like, this PR breaks file ownership either way: Previously the files were owned by `nogroup`, now they'll be owned by `rtkit` (whether it be dynamically allocated or statically).

---

ok but then I should also remove the static uid

---

Some more info: We discussed this on matrix. @symphorien made a good argument that whether we choose static or dynamic ids, it should be the same for uid and gid, because any reason to use either static or dynamic can equally be applied to both uid's and gid's.

I suggested to then make them both dynamic, in order to go towards the RFC's direction

---

done.